.. fillintheblank:: return_main_2
    :author: bmiller
    :difficulty: 3.0
    :basecourse: thinkcpp
    :chapter: Chapter5
    :subchapter: returningfrommain
    :topics: Chapter5/returningfrommain
    :from_source: T
    :pct_on_first: 0.5
    :total_students_attempting: 2
    :num_students_correct: 2.0
    :mean_clicks_to_correct: 1.5

    Usually, we return |blank| to exit main.  However, if there are
    any errors that we catch, we might return |blank|.
    
    - :[0]: Correct!
      :x: Try again!
    - :-1: Correct!
      :.*: Try again!